src/utils/iced.rs

```diff
@@ -251,13 +253,16 @@ impl<P: Program + Send + 'static> IcedElement<P> {
             })
             .ok();
 
+        let theme_guard = THEME.read().unwrap();
+        let theme = theme_guard.clone();
+        drop(theme_guard);
```

Comment:
Can't this simply be?

```suggestion
        let theme = THEME.read().unwrap().clone();
```